cdap-explore/src/main/java/co/cask/cdap/explore/service/ExploreServiceUtils.java

```diff
@@ -64,9 +64,13 @@
    */
   public enum HiveSupport {
     // CDH 5.0 to 5.1 uses Hive 0.12
-    // CDH >5.1 uses Hive >=0.13.1 (aka 1.0, which Hive14ExploreService supports)
+    // The order of the enum values below is very important
     HIVE_CDH5_0(Pattern.compile("^.*cdh5.0\\..*$"), Hive12CDH5ExploreService.class),
     HIVE_CDH5_1(Pattern.compile("^.*cdh5.1\\..*$"), Hive12CDH5ExploreService.class),
+    // CDH 5.2.x and 5.3.x use Hive 0.13
+    HIVE_CDH5_2(Pattern.compile("^.*cdh5.2\\..*$"), Hive13ExploreService.class),
+    HIVE_CDH5_3(Pattern.compile("^.*cdh5.3\\..*$"), Hive13ExploreService.class),
+    // CDH >5.3 uses Hive >=0.14.1 (which Hive14ExploreService supports)
```

Comment:
Should be `Hive >=1.1.0` http://blog.cask.co/2015/06/hadoop-components-versions-in-distros-matrix/